source/index.html.md

```diff
@@ -309,3 +309,28 @@ Always pass in the most up-to-date value for <code>total_paid</code> as a user's
 <aside class='notice'>
   Updates to orders will not be saved until the user who created the order is "authenticated" via the SDK. In practice, this means that a <code>ticketingOrder.update</code> command will be processed immediately for users who have been "autenticated" by a previous <code>emailSignup</code> or <code>phoneNumberSignup</code> command. For users who weren't "authenticated" before a <code>ticketingOrder.update</code> command is made, the <code>ticketingOrder.update</code> command will be "buffered" by the SDK and processed immediately following a <code>emailSignup</code> or <code>phoneNumberSignup</code> command (which can potentially happen in the future on a separate pageview or session entirely).
 </aside>
+
+# Add Custom User Properties
+
+This command will add and update custom user properties for the current user. 
+
+```javascript
```

Comment:
i believe we need to add this to add a header for the code block!

![image](https://user-images.githubusercontent.com/4978606/198380988-adb50dfd-5222-4f66-b64d-531c5f13a87b.png)

```suggestion
> Use the following command to set custom user properties on the authenticated user.

```javascript
```